meta/shared/src/main/scala/scalaz/meta/Ops.scala

```diff
@@ -39,6 +39,10 @@ class Ops(val c: blackbox.Context) {
   def n_0[T: c.WeakTypeTag]: Tree            = q"$ev.${typeName[T]}($lhs)"
   def ni_0[T: c.WeakTypeTag](ev: Tree): Tree = q"$ev.${typeName[T]}($lhs)"
 
+  /* def method(implicit g: B): R */
```

Comment:
`def method(lhs: T)(implicit g: B): R`?

Reply:
I really want to get rid of all these macros and make scalaz-plugin just inline syntax all the time.